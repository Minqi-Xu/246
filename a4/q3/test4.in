new 8
play 2 5
play 2 2
play 5 2
play 5 5
play 2 3
play 4 2
play 0 1
play 7 2
play 0 2
play 3 5
play 0 3
play 3 2
play 0 4
play 0 0
play 0 5
play 6 6
play 0 6
play 1 6
play 1 1
play 0 7
play 7 7
