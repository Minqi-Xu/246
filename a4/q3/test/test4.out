--------
--------
--------
---BW---
---WB---
--------
--------
--------
--------
--------
-----B--
---BW---
---WB---
--------
--------
--------
--------
--------
--W--B--
---BW---
---WB---
--------
--------
--------
--------
--------
--W--B--
---BB---
---BB---
--B-----
--------
--------
--------
--------
--W--B--
---WB---
---BW---
--B--W--
--------
--------
--------
--------
--WB-B--
---BB---
---BW---
--B--W--
--------
--------
--------
--------
--WB-B--
---BB---
--WWW---
--B--W--
--------
--------
-B------
--------
--WB-B--
---BB---
--WWW---
--B--W--
--------
--------
-B------
--------
--WB-B--
---BB---
--WWW---
--B--W--
--------
--W-----
-BB-----
--------
--WB-B--
---BB---
--WWW---
--B--W--
--------
--W-----
-BB-----
--------
--WB-B--
---BBW--
--WWW---
--B--W--
--------
--W-----
-BBB----
--------
--WB-B--
---BBW--
--WWW---
--B--W--
--------
--W-----
-BBB----
--------
--WB-B--
--WWWW--
--WWW---
--B--W--
--------
--W-----
-BBBB---
--------
--WB-B--
--WWWW--
--WWW---
--B--W--
--------
--W-----
WBBBB---
--------
--WB-B--
--WWWW--
--WWW---
--B--W--
--------
--W-----
WBBBBB--
--------
--WB-B--
--WWWW--
--WWW---
--B--W--
--------
--W-----
WBBBBB--
--------
--WB-B--
--WWWW--
--WWW---
--B--W--
------W-
--W-----
WBBBBBB-
--------
--WB-B--
--WWWW--
--WWW---
--B--W--
------W-
--W-----
WBBBBBB-
------W-
--WB-W--
--WWWW--
--WWW---
--B--W--
------W-
--W-----
WBBBBBB-
-B----W-
--WB-W--
--WWWW--
--WWW---
--B--W--
------W-
--W-----
WWWWWWWW
-B----W-
--WB-W--
--WWWW--
--WWW---
--B--W--
------W-
--W-----
WWWWWWWW
-B----W-
--BB-W--
--WBWW--
--WWB---
--B--B--
------B-
--W----B
